sdk/samples/iot/docs/how_to_iot_hub_samples_windows.md

```diff
@@ -155,13 +155,15 @@ To run the samples, ensure you have the following programs and tools installed o
     PS C:\azure-sdk-for-c\sdk\samples\iot> openssl x509 -noout -fingerprint -in device_ec_cert.pem | % {$_.replace(":", "")} | % {$_.replace("SHA1 Fingerprint=", "")} | Tee-Object fingerprint.txt
     ```
 
-    <details><summary><i>The output will be the fingerprint (also stored in `fingerprint.txt`) and will look similar to:</i></summary>
-    <p?>
```

Comment:
How'd that get in there?

Reply:
Apparently I added it in one of my last PRs...